src/registry.js

```diff
@@ -286,7 +286,7 @@ function registerTraceModule(_module) {
 
     // add maplibre-gl CSS here to avoid console warning on instantiation
     if(bpmName === 'map') {
-        addCss('https://unpkg.com/maplibre-gl@^4.3.2/dist/maplibre-gl.css');
+        addCss('/node_modules/maplibre-gl/dist/maplibre-gl.css');
```

Comment:
I think you should `require` this file.

Reply:
@birkskyum Wondering perhaps we need to add it similar to the way it is done for mapbox.
So instead of `addCss` function you may use these functions:
https://github.com/plotly/plotly.js/blob/e1ca2a8160f85ef6122c842646f25fa59c10784e/src/lib/dom.js#L52-L77

Reply:
I think you're very right - it appears to work with requiring the css, and adding the style-loader and css-loader to webpack.

We can add the `esbuild-style-plugin` too, to make it work there as well.